pkg/monitors/http/http.go

```diff
@@ -214,15 +213,20 @@ func (m *Monitor) getTLSStats(site *url.URL, logger *logrus.Entry) (dps []*datap
 		serverName = host
 	}
 
+	dimensions := map[string]string{
+		"server_name":     host,
+		"sni_server_name": serverName,
+	}
+
 	ipConn, err := net.Dial("tcp", host+":"+port)
 	if err != nil {
+		logger.WithError(err).Error("connection failed to host during TLS stat collection")
 		return
 	}
 	defer ipConn.Close()
 
 	tlsCfg := &tls.Config{
-		InsecureSkipVerify: m.conf.SkipVerify,
```

Comment:
I think the point of this originally is so that you can validate the time left on a cert even if it is issued by a private CA that the agent doesn't have in its cert store.

So I would just leave this.

Reply:
If I leave it as is, then I won't be able to use the client to do the cert verification as  `m.conf.SkipVerify` is getting set to true.
If the cert is issued by a private CA, isn't the expectation that user will set the CA certs `auth.TLSConfig(tlsCfg, m.conf.CACertPath, m.conf.ClientCertPath, m.conf.ClientKeyPath)`  which will make the handshake to pass and the time left on cert successfully  calculated?

Reply:
ok, yeah I guess if it is marked as invalid in the monitor it is less important that the time until expiry get set properly.